src/types/task.list.ts

```diff
@@ -50,11 +52,11 @@ export interface TaskList {
   csDue: boolean;
 }
 
-export const initTaskList = (): TaskList => {
+export const initTaskList = (companyNumber: string): TaskList => {
   return {
     tasks: {
       officers: { isVisible: false, state: TaskState.NOT_CHECKED },
-      peopleSignificantControl: { isVisible: false, state: TaskState.NOT_CHECKED },
+      peopleSignificantControl: { isVisible: false, state: TaskState.NOT_CHECKED, url: addComapnyNumber(companyNumber, ACTIVE_PSCS_PATH) },
```

Comment:
you can use that new url function here as well - https://github.com/companieshouse/confirmation-statement-web/blob/main/src/utils/url.ts#L3

Reply:
Updated this with a merge from main.